platform_channels/lib/src/image_basic_message_channel.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:typed_data';
+import 'package:flutter/services.dart';
+
+/// This class includes the implementation for [BasicMessageChannel] to get
+/// a platform image. The [BasicMessageChannel] is using [StandardMessageCodec]
+/// since it supports [Uint8List] which can be easily used to get the image data
+/// from platform and display an image using [Image.memory].
```

Comment:
nit: /// This class manages a [BasicMessageChannel] that can return an image loaded
/// from a native asset. The [BasicMessageChannel] uses [StandardMessageCodec]
/// since it supports [Uint8List], which is used to transport the image data.